assets/js/theme/common/carousel/utils/setupAriaAttributes.js

```diff
@@ -0,0 +1,11 @@
+export default ($node, ariaAttribute, attributeValue, ...accessibiltyClassList) => {
+    const accessibilityTextContainer = `<span class="${accessibiltyClassList.join(' ')}"></span>`;
+    const $accessibilityElement = $node.find(`.${accessibiltyClassList[0]}`);
+
+    if ($accessibilityElement.length) {
+        $accessibilityElement.attr(ariaAttribute, attributeValue);
+    } else {
+        const $accessibilityElemTweak = $(accessibilityTextContainer).attr(ariaAttribute, attributeValue);
+        $node.append($accessibilityElemTweak);
```

Comment:
Could we replace inserting nodes with using visually hidden classes?

Reply:
removed setupAriaAttributes util at all.